Source/APIClient/Request.swift

```diff
@@ -19,6 +19,7 @@ public protocol Request {
     static var baseURL: String { get }
     static var acceptHeader: String? { get }
     static var authToken: String? { get }
+    static var authTokenHeader: String? { get }
```

Comment:
I think since we're exposing the entire header value we should probably just use a single property instead of splitting it into the prefix and token. The user would then be responsible for defining the string format.

I like the suggestion of naming it `authorizationHeader`, since it's the value of the `Authorization` key.